### DISCUSSION TITLE
Using tree rings and shipwrecks to track historical hurricanes in the Caribbean

### DISCUSSION THREAD

**Discussion by earthlab-test-student** [earthlab-education/Earth-Analytics-2023-01-Intro#10] [Awesome science]
Check out [this podcast from Radiolab](https://radiolab.org/podcast/fellowship-tree-rings) about compiling an historical hurricane record using tree ring data and shipwreck logs.

**Comment by megan8617** [comment]
Love 99pi. My favorite episode is Ten Thousand Years and the glow-in-the-dark cats: https://99percentinvisible.org/episode/ten-thousand-years/
